mapreader/download/downloader.py

```diff
@@ -1,11 +1,16 @@
-from typing import Union, Optional
+from __future__ import annotations
```

Comment:
This import crops up in a few files, is it what's causing the merge conflict?

Reply:
I wanted to add [pyupgrade](https://github.com/asottile/pyupgrade) to my ruff rules so that then I could completely get rid of Union/Optional/etc so one of the setting is to always add this import. Will just add and run this now.